armory/attacks/librispeech_target_labels.py

```diff
@@ -0,0 +1,544 @@
+# ground truth labels for first 100 samples in the test set
```

Comment:
we discussed in the meeting whether this is the best location for this. I thought it seemed a bit awkward since it's not an attack, per se, and it's now the only thing living in this dir. You had already considered placing it in `armory/utils/labels.py` and opted against that, correct?

Reply:
Correct. I think the data portion of `librispeech_target_labels.py` and code portion of `labels.py` should be separate. I think that `labels.py` should actually be moved out of utils at some point, but don't want to modify too much at present.